lightkurve/targetpixelfile.py

```diff
@@ -374,6 +384,48 @@ def to_lightcurve(self, method='aperture', **kwargs):
         else:
             raise ValueError("Photometry method must be 'aperture' or 'prf'.")
 
+    def extract_aperture_photometry(self, aperture_mask='pipeline'):
+        """Returns a LightCurve obtained using aperture photometry.
+
+        Parameters
+        ----------
+        aperture_mask : array-like, 'pipeline', 'threshold' or 'all'
+            A boolean array describing the aperture such that `True` means
+            that the pixel will be used.
+            If None or 'all' are passed, all pixels will be used.
+            If 'pipeline' is passed, the mask suggested by the official pipeline
+            will be returned.
+            If 'threshold' is passed, all pixels brighter than 3-sigma above
+            the median flux will be used.
+
+        Returns
+        -------
+        lc : KeplerLightCurve object
```

Comment:
Suggested change to fix the return type and render nicely:
```suggestion
        lc : `.LightCurve` object
```